src/main/kotlin/uk/gov/gdx/datashare/service/DeathNotificationService.kt

```diff
@@ -145,3 +145,9 @@ data class DeathNotificationDetails(
   @Schema(description = "Whether the deceased was retired", required = false, example = "false")
   val retired: Boolean? = null,
 )
+
+enum class GenderType {
```

Comment:
```suggestion
enum class Gender {
```
not sure what "type" adds here, but happy to discuss

Reply:
@olevett updated